hedera-node/hedera-app/build.gradle.kts

```diff
@@ -63,3 +63,91 @@ dependencies {
 tasks.withType<Test> {
     testLogging.exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
 }
+
+// Add all the libs dependencies into the jar manifest!
+tasks.jar {
+    doFirst {
+        tasks.jar.configure {
+            manifest {
+                attributes(
+                    "Main-Class" to "com.hedera.node.app.ServicesMain",
+                    "Class-Path" to configurations.getByName("runtimeClasspath")
+                        .joinToString(separator = " ") { "../../data/lib/" + it.name }
+
+                )
+            }
+        }
+    }
+}
+
+// Replace variables in semantic-version.properties with build variables
+tasks.processResources {
+    filesMatching("semantic-version.properties") {
+        filter { line: String ->
+            if (line.contains("hapi-proto.version")) {
+                "hapi.proto.version=" + libs.versions.hapi.version.get()
+            } else if (line.contains("project.version")) {
+                "hedera.services.version=" + project.version
+            } else {
+                line
+            }
+        }
+    }
+}
+
+// Copy dependencies into `data/lib`
+val copyLib = tasks.register<Copy>("copyLib") {
+    from(project.configurations.getByName("runtimeClasspath"))
+    into(project(":hedera-node").file("data/lib"))
```

Comment:
can we move this into the build folder?

Reply:
I would like to, but I think that is a different PR. This one doesn't propose to change where these files end up.

Reply:
you are right. I assumed you added this code. Have not seen that it was just moved.